internal/lightcone/destruction/theunreachableside/theunreachableside.go

```diff
@@ -0,0 +1,84 @@
+package theunreachableside
+
+import (
+	"github.com/simimpact/srsim/pkg/engine"
+	"github.com/simimpact/srsim/pkg/engine/equip/lightcone"
+	"github.com/simimpact/srsim/pkg/engine/event"
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/modifier"
+	"github.com/simimpact/srsim/pkg/engine/prop"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+const (
+	side    key.Modifier = "the-unreachable-side"
+	dmgBuff key.Modifier = "the-unreachable-side-dmg-buff"
+)
+
+// Increases the wearer's CRIT rate by 18% and increases their Max HP by 18%.
+// When the wearer is attacked or consumes their own HP, their DMG increases by 24%.
+// This effect is removed after the wearer uses an attack.
+
+func init() {
+	lightcone.Register(key.TheUnreachableSide, lightcone.Config{
+		CreatePassive: Create,
+		Rarity:        5,
+		Path:          model.Path_DESTRUCTION,
+		Promotions:    promotions,
+	})
+	modifier.Register(side, modifier.Config{
+		Listeners: modifier.Listeners{
+			OnAfterBeingAttacked: buffOnAttacked,
+			OnHPChange:           buffOnHPConsume,
+			OnAfterAttack:        removeBuff,
+		},
+	})
+	modifier.Register(dmgBuff, modifier.Config{
+		Stacking: modifier.ReplaceBySource,
+	})
```

Comment:
* `StatusType BUFF`
* Stacking Behavior should be `Unique`/default and not `ReplaceBySource`